test/src/rabbit_auth_backend_ldap_test.erl

```diff
@@ -100,12 +100,18 @@ login() ->
 
 logins() ->
     [{bad, #amqp_params_network{}},
-     {bad, #amqp_params_network{username = <<"Simon MacMullen">>}},
-     {bad, #amqp_params_network{username = <<"Simon MacMullen">>,
+     {bad, #amqp_params_network{username = << ?SIMON_NAME >>}},
```

Comment:
I suggest that we switch to using Alice and Bob instead of former or current team member names.